Release

evereq opened the pull request:
### Before submitting the PR, please make sure you do the following

 1. Contributor license agreement
For us it's important to have the agreement of our contributors to use their work, whether it be code or documentation. Therefore, we are asking all contributors to sign a contributor license agreement (CLA) as commonly accepted in most open source projects. **Just open the pull request and our CLA bot will prompt you briefly.**

 2. Please check our [contribution guidelines](https://docs.traduora.co/docs/contributing#review-process-for-this-repo) for some help in the process.

evereq committed:
Update README.md

vvadymk committed:
fix: change progress rounding to down

evereq committed:
Merge pull request #387 from vvadymk/fix/progress-rounding

fix: change translation progress rounding to down

vvadymk committed:
Feat: add export with untranslated keys (#389)

* feat: add export with untranslated keys

* fix: check with prettier

* fix: fix lint error

* fix: edit e2e tests for untranslated feature

* fix: fix tests

* fix: fix column name in query

* fix: naming of the projectId to camelCase

* fix: fix tests

evereq committed:
Update README.md

evereq committed:
Create CODE_OF_CONDUCT.md

evereq committed:
Create CODE_OF_CONDUCT.md

evereq committed:
Delete CODE_OF_CONDUCT.md

evereq committed:
Update README.md

vvadymk committed:
Feat: add postgresql as db (#391)

* feat: add postgresql as db

* feat: add ability to work both with pgsql and mysql

* fix: add break in switch

* change config

* fix: fix entities type

evereq committed:
chore: add docker compose with PostgreSQL

evereq committed:
chore: a bit more logging

evereq committed:
chore: formatting

evereq committed:
chore: lint

evereq committed:
Update README.md

dependabot[bot] committed:
build(deps): bump @babel/traverse from 7.15.4 to 7.23.2

Bumps [@babel/traverse](https://github.com/babel/babel/tree/HEAD/packages/babel-traverse) from 7.15.4 to 7.23.2.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.23.2/packages/babel-traverse)

---
updated-dependencies:
- dependency-name: "@babel/traverse"
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

evereq committed:
Merge pull request #393 from ever-co/dependabot/npm_and_yarn/babel/traverse-7.23.2

build(deps): bump @babel/traverse from 7.15.4 to 7.23.2

dependabot[bot] committed:
build(deps): bump word-wrap from 1.2.3 to 1.2.5

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.5.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.5)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

evereq committed:
Merge pull request #394 from ever-co/dependabot/npm_and_yarn/word-wrap-1.2.5

build(deps): bump word-wrap from 1.2.3 to 1.2.5

dependabot[bot] committed:
build(deps): bump follow-redirects from 1.14.8 to 1.15.6

Bumps [follow-redirects](https://github.com/follow-redirects/follow-redirects) from 1.14.8 to 1.15.6.
- [Release notes](https://github.com/follow-redirects/follow-redirects/releases)
- [Commits](https://github.com/follow-redirects/follow-redirects/compare/v1.14.8...v1.15.6)

---
updated-dependencies:
- dependency-name: follow-redirects
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump typeorm from 0.2.37 to 0.3.0 in /api

Bumps [typeorm](https://github.com/typeorm/typeorm) from 0.2.37 to 0.3.0.
- [Release notes](https://github.com/typeorm/typeorm/releases)
- [Changelog](https://github.com/typeorm/typeorm/blob/master/CHANGELOG.md)
- [Commits](https://github.com/typeorm/typeorm/compare/0.2.37...0.3.0)

---
updated-dependencies:
- dependency-name: typeorm
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump typeorm from 0.2.37 to 0.3.0

Bumps [typeorm](https://github.com/typeorm/typeorm) from 0.2.37 to 0.3.0.
- [Release notes](https://github.com/typeorm/typeorm/releases)
- [Changelog](https://github.com/typeorm/typeorm/blob/master/CHANGELOG.md)
- [Commits](https://github.com/typeorm/typeorm/compare/0.2.37...0.3.0)

---
updated-dependencies:
- dependency-name: typeorm
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

evereq committed:
Merge pull request #392 from ever-co/feat/postgres

Feat/postgres

evereq committed:
Merge pull request #396 from ever-co/dependabot/npm_and_yarn/follow-redirects-1.15.6

build(deps): bump follow-redirects from 1.14.8 to 1.15.6

evereq committed:
Merge pull request #397 from ever-co/dependabot/npm_and_yarn/api/typeorm-0.3.0

build(deps): bump typeorm from 0.2.37 to 0.3.0 in /api

evereq committed:
Merge pull request #398 from ever-co/dependabot/npm_and_yarn/typeorm-0.3.0

build(deps): bump typeorm from 0.2.37 to 0.3.0

rahul-rocket committed:
fix: typeorm repository types

rahul-rocket committed:
chore(deps): bump @nestjs/core from ^8.0.6 to ^10.3.8

evereq committed:
Merge pull request #400 from ever-co/fix/test-e2e

[Fix] Test e2e

dependabot[bot] committed:
build(deps): bump express from 4.17.1 to 4.19.2

Bumps [express](https://github.com/expressjs/express) from 4.17.1 to 4.19.2.
- [Release notes](https://github.com/expressjs/express/releases)
- [Changelog](https://github.com/expressjs/express/blob/master/History.md)
- [Commits](https://github.com/expressjs/express/compare/4.17.1...4.19.2)

---
updated-dependencies:
- dependency-name: express
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

evereq committed:
chore: use Node 16.20.2

evereq committed:
chore: formatting

dependabot[bot] committed:
build(deps): bump mysql2 from 3.9.6 to 3.9.7

Bumps [mysql2](https://github.com/sidorares/node-mysql2) from 3.9.6 to 3.9.7.
- [Release notes](https://github.com/sidorares/node-mysql2/releases)
- [Changelog](https://github.com/sidorares/node-mysql2/blob/master/Changelog.md)
- [Commits](https://github.com/sidorares/node-mysql2/compare/v3.9.6...v3.9.7)

---
updated-dependencies:
- dependency-name: mysql2
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump ejs from 3.1.7 to 3.1.10 in /api

Bumps [ejs](https://github.com/mde/ejs) from 3.1.7 to 3.1.10.
- [Release notes](https://github.com/mde/ejs/releases)
- [Commits](https://github.com/mde/ejs/compare/v3.1.7...v3.1.10)

---
updated-dependencies:
- dependency-name: ejs
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump ejs from 3.1.7 to 3.1.10

Bumps [ejs](https://github.com/mde/ejs) from 3.1.7 to 3.1.10.
- [Release notes](https://github.com/mde/ejs/releases)
- [Commits](https://github.com/mde/ejs/compare/v3.1.7...v3.1.10)

---
updated-dependencies:
- dependency-name: ejs
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump ws from 6.2.2 to 6.2.3

Bumps [ws](https://github.com/websockets/ws) from 6.2.2 to 6.2.3.
- [Release notes](https://github.com/websockets/ws/releases)
- [Commits](https://github.com/websockets/ws/compare/6.2.2...6.2.3)

---
updated-dependencies:
- dependency-name: ws
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
build(deps): bump socket.io from 4.5.1 to 4.7.5

Bumps [socket.io](https://github.com/socketio/socket.io) from 4.5.1 to 4.7.5.
- [Release notes](https://github.com/socketio/socket.io/releases)
- [Changelog](https://github.com/socketio/socket.io/blob/main/CHANGELOG.md)
- [Commits](https://github.com/socketio/socket.io/compare/4.5.1...4.7.5)

---
updated-dependencies:
- dependency-name: socket.io
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

evereq committed:
Merge pull request #403 from ever-co/dependabot/npm_and_yarn/mysql2-3.9.7

build(deps): bump mysql2 from 3.9.6 to 3.9.7

evereq committed:
Merge pull request #414 from ever-co/dependabot/npm_and_yarn/socket.io-4.7.5

build(deps): bump socket.io from 4.5.1 to 4.7.5

evereq committed:
Merge pull request #413 from ever-co/dependabot/npm_and_yarn/ws-6.2.3

build(deps): bump ws from 6.2.2 to 6.2.3

evereq committed:
Merge pull request #407 from ever-co/dependabot/npm_and_yarn/ejs-3.1.10

build(deps): bump ejs from 3.1.7 to 3.1.10

evereq committed:
Merge pull request #406 from ever-co/dependabot/npm_and_yarn/api/ejs-3.1.10

build(deps): bump ejs from 3.1.7 to 3.1.10 in /api

evereq committed:
Merge pull request #401 from ever-co/dependabot/npm_and_yarn/express-4.19.2

build(deps): bump express from 4.17.1 to 4.19.2

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ever-co/ever-traduora?pullRequest=416) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ever-co/ever-traduora?pullRequest=416) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: evereq<br/>:white_check_mark: vvadymk<br/>:white_check_mark: rahul-rocket<br/>:x: dependabot[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ever-co/ever-traduora?pullRequest=416) it.</sub>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ever-co/ever-traduora?pullRequest=416) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ever-co/ever-traduora?pullRequest=416) before we can accept your contribution.<br/>**3** out of **4** committers have signed the CLA.<br/><br/>:white_check_mark: evereq<br/>:white_check_mark: vvadymk<br/>:white_check_mark: rahul-rocket<br/>:x: dependabot[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ever-co/ever-traduora?pullRequest=416) it.</sub>

evereq committed:
chore: update to support Node 20 and fix docker compose

rahul-rocket committed:
[Chore] Nest Packages Updated (#417)

* chore(deps): bump @nestjs/core from ^10.3.8 to ^10.3.9

* chore(deps): bump typescript from ^4.2.3 to ^5.5.2

* chore(deps): bump dev dependencies

* fix: run web and api start command from root package

* fix: updated package names

---------

Co-authored-by: Ruslan Konviser <evereq@gmail.com>

dependabot[bot] committed:
build(deps): bump pug from 3.0.2 to 3.0.3

Bumps [pug](https://github.com/pugjs/pug) from 3.0.2 to 3.0.3.
- [Release notes](https://github.com/pugjs/pug/releases)
- [Commits](https://github.com/pugjs/pug/compare/pug@3.0.2...pug@3.0.3)

---
updated-dependencies:
- dependency-name: pug
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

evereq committed:
chore: update yarn.lock

evereq committed:
Merge pull request #419 from ever-co/dependabot/npm_and_yarn/pug-3.0.3

build(deps): bump pug from 3.0.2 to 3.0.3

evereq committed:
Update README.md

evereq committed:
Update config.yml

rahul-rocket committed:
[Fix] Jest Test e2e (#402)

* chore(deps): bump jest from ^27.1.1 to ^29.7.0

* fix: updated gitignore file

* fix: typeorm nested relations query syntax

* chore(deps): bump prettier from ^2.4.0 to ^3.2.5

* fix:  run prettier for formatting

* fix: run prettier for formatting

* fix: typeorm nested relations query syntax

* fix: get data source connection for e2e testing

* fix: run prettier for formatting

* fix: run prettier for formatting

* fix: increased jest testTimeout

* fix: removed maxWorker for jest

* update yarn.lock

* chore: update yarn.lock

* fix: coderabbitai suggestion

* fix: coderabbitai suggestion

* chore: update yarn.lock

* fix: removed console

* style: fix code style issues using Prettier

* fix: updated connection datasource

evereq committed:
fix: fixing Jest tests to ignore space formatting

evereq committed:
chore: formatting

evereq committed:
fix: trying another jest fix

evereq committed:
fix: attempt to fix Jest tests

evereq committed:
fix: slow down Jest to see if it helps with testing issues

rahul-rocket committed:
[Chore] Upgrade Angular From v12 to v16 (#423)

* chore: format code with Prettier

* build(deps): bump angular/core from ^12.2.5 to ^13.4.0

* build(deps): bump ng-bootstrap from ^10.0.0 to ^12.1.2

* update yarn.lock

* build(deps): bump angular/core from ^13.4.0 to ^14.3.0

* build(deps): bump angular/core from ^14.3.0 to ^15.2.10

* build(deps): bump angular/core from ^15.2.10 to ^16.2.12

* build(deps): bump ng-bootstrap from ^14.2.0 to ^15.1.2

* chore: update NGXS packages to version 3.8.2

- Updated @ngxs/store, @ngxs/devtools-plugin, and @ngxs/router-plugin to 3.8.2

* style: fix code style issues using Prettier

* chore: add lint target to angular.json for ESLint integration

* updated angular.json

* updated angular.json

* [Chore] Add --open option to start script for automatic browser launch

* build(deps): bump bootstrap from ^4.6.2 to ^5.1.1

* build(deps): downgraded bootstrap from ^5.1.1 to ^4.6.2

rahul-rocket committed:
build(deps-dev): bump webpack from 5.93.0 to 5.94.0

rahul-rocket committed:
build(deps-dev): bump webpack from 5.94.0 to 5.95.0

rahul-rocket committed:
build(deps-dev): @types/express webpack from 4.17.21 to 5.0.0

rahul-rocket committed:
build(deps-dev): bump @nestjs/core from 10.3.9 to 10.4.6

rahul-rocket committed:
Merge pull request #424 from ever-co/chore/webpack-5.94.0

build(deps-dev): bump webpack from 5.93.0 to 5.95.0

rahul-rocket committed:
chore(deps): remove unnecessary crypto dependency

rahul-rocket committed:
Merge branch 'develop' into chore/update-packages

rahul-rocket committed:
Merge pull request #426 from ever-co/chore/remove-crypto-dependency

chore(deps): remove unnecessary crypto dependency

rahul-rocket committed:
Merge branch 'develop' into chore/update-packages

rahul-rocket committed:
Update yarn.lock

rahul-rocket committed:
Merge pull request #425 from ever-co/chore/update-packages

[Chore] Update Packages

rahul-rocket committed:
refactor: replace UntypedFormBuilder with FormBuilder

rahul-rocket committed:
Merge pull request #427 from ever-co/untyped-form-builder

Refactor: Replace UntypedFormBuilder with FormBuilder

evereq committed:
chore: linting

evereq committed:
chore: prettier

evereq committed:
chore: prettier
